test/integration/users.test.js

```diff
@@ -1229,4 +1231,80 @@ describe("Users", function () {
         });
     });
   });
+
+  describe("POST /users/migrate", function () {
+    let fetchStub;
+
+    beforeEach(async function () {
+      fetchStub = Sinon.stub(axios, "get");
+    });
+
+    afterEach(async function () {
+      Sinon.restore();
+    });
+
+    it("Should update the user", async function () {
+      fetchStub.resolves({
+        data: githubUserInfo[0]._json,
+      });
+      const usersMigrateResponse = await chai
+        .request(app)
+        .post(`/users/migrate`)
+        .set("Cookie", `${cookieName}=${superUserAuthToken}`);
+      expect(usersMigrateResponse).to.have.status(200);
+      expect(usersMigrateResponse.body).to.eql({
```

Comment:
please use to deep equal or strict equal

Reply:
Updated the code